frontend/src/containers/TrimPage/TrimBannerContainer.tsx

```diff
@@ -24,36 +24,38 @@ export default function TrimBannerContainer() {
   return (
     <Banner subtitle={'기본에 충실한 펠리세이드'} title={'Le blanc'}>
       <Container>
-        <HmgDataSection>
-          <HmgTag size="small" />
-          <HmgTagDescription>
-            해당 트림 포함된 옵션들의
-            <BlueText> 실활용 데이터</BlueText>예요.
-          </HmgTagDescription>
-          <DataList>
-            <Data>
-              <DataTitle>안전 하차 보조</DataTitle>
-              <DataInfo>
-                42회
-                <DataCaption>15,000km 당</DataCaption>
-              </DataInfo>
-            </Data>
-            <Data>
-              <DataTitle>후측방 충돌 경고</DataTitle>
-              <DataInfo>
-                42회
-                <DataCaption>15,000km 당</DataCaption>
-              </DataInfo>
-            </Data>
-            <Data>
-              <DataTitle>후방 교차 충돌방지 경고</DataTitle>
-              <DataInfo>
-                42회
-                <DataCaption>15,000km 당</DataCaption>
-              </DataInfo>
-            </Data>
-          </DataList>
-        </HmgDataSection>
+        <HmgDataWrapper style={{ zIndex: 10000 }}>
```

Comment:
```suggestion
        <HmgDataWrapper >
```
style={{ zIndex: 10000 }} 부분은 styled component 안에 넣어주세요

Reply:
네이